HPXMLtoOpenStudio/resources/EPvalidator.rb

```diff
@@ -136,7 +136,7 @@ def self.run_validator(hpxml_doc)
       # [Wall]
       "/HPXML/Building/BuildingDetails/Enclosure/Walls/Wall" => {
         "SystemIdentifier" => one, # Required by HPXML schema
-        "[ExteriorAdjacentTo='outside' or ExteriorAdjacentTo='attic - vented' or ExteriorAdjacentTo='attic - unvented' or ExteriorAdjacentTo='basement - conditioned' or ExteriorAdjacentTo='basement - unconditioned' or ExteriorAdjacentTo='crawlspace - vented' or ExteriorAdjacentTo='crawlspace - unvented' or ExteriorAdjacentTo='garage' or ExteriorAdjacentTo='other housing unit']" => one,
+        "[ExteriorAdjacentTo='outside' or ExteriorAdjacentTo='attic - vented' or ExteriorAdjacentTo='attic - unvented' or ExteriorAdjacentTo='basement - conditioned' or ExteriorAdjacentTo='basement - unconditioned' or ExteriorAdjacentTo='crawlspace - vented' or ExteriorAdjacentTo='crawlspace - unvented' or ExteriorAdjacentTo='garage' or ExteriorAdjacentTo='other housing unit' or ExteriorAdjacentTo='other heated space' or ExteriorAdjacentTo='other multifamily buffer space' or ExteriorAdjacentTo='other non-freezing space']" => one,
```

Comment:
Do we want to also support foundation walls and rim joists to be "ExteriorAdjacentTo" these new space types?

Reply:
Yes, as well as FrameFloors.